.github/workflows/operatorBuildAndDeploy.yml

```diff
@@ -38,35 +38,143 @@ jobs:
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+      - name: Build and Push Docker
+        if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == 'datastax/cass-operator'
+        env:
+          ECR_REPO: ${{ secrets.ECR_REPO }}
           PR_REF: ${{ github.event.pull_request.head.ref }}
-          MO_BASE_OS: 'registry.access.redhat.com/ubi7/ubi-minimal:7.8'
         run: |
           if [ "${GITHUB_EVENT_NAME}" == "pull_request" ]; then
             export MO_BRANCH=${PR_REF}
           else
             export MO_BRANCH="master"
-          fi;
-          export PATH=$GOROOT/bin:$GOPATH/bin:$PATH
-          mage operator:testAndBuild
-      - name: Deploy to ECR
-        if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == 'datastax/cass-operator'
-        env:
-          MO_ECR_ID: ${{ secrets.ECR_ID }}
-          MO_ECR_SECRET: ${{ secrets.ECR_SECRET }}
-          MO_ECR_REPO: ${{ secrets.ECR_REPO }}
-        run: |
-          export PATH=$GOROOT/bin:$GOPATH/bin:$PATH
-          export MO_TAGS=$(cat ./build/tagsToPush.txt)
-          mage operator:deployToECR
-      - name: Deploy to GH Packages
-        if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == 'datastax/cass-operator'
-        env:
-          MO_GH_USR: 'datastax/cass-operator'
-          MO_GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          MO_GH_PKG_REPO: 'datastax/cass-operator/operator'
-        run: |
+          fi
+
           export PATH=$GOROOT/bin:$GOPATH/bin:$PATH
-          export MO_TAGS=$(cat ./build/tagsToPush.txt)
-          mage operator:deployToGHPackages
+          GITHUB_REPO_URL="https://github.com/${{ github.repository }}"
+
+          VERSION="$(mage operator:printVersion)"
+          FULL_VERSION="$(mage operator:printFullVersion)"
+          VERSION_STAMP="${{ github.repository }}:${FULL_VERSION}"
+
+          ECR_REPOSITORY="${ECR_REPO}/datastax/cass-operator"
+          GH_REPOSITORY="docker.pkg.github.com/${{ github.repository_owner }}/cass-operator/operator"
+
+          ECR_TAGS=()
+          ECR_UBI_TAGS=()
+          GH_TAGS=()
+          GH_UBI_TAGS=()
+          GH_ARM64_TAGS=()
+
+          for t in "${FULL_VERSION}" "${GITHUB_SHA}" "latest"; do
+            ECR_TAGS+=(--tag "${ECR_REPOSITORY}:${t}")
+            ECR_UBI_TAGS+=(--tag "${ECR_REPOSITORY}:${t}-ubi")
+
+            GH_TAGS+=(--tag "${GH_REPOSITORY}:${t}")
+            GH_UBI_TAGS+=(--tag "${GH_REPOSITORY}:${t}-ubi")
+            GH_ARM64_TAGS+=(--tag "${GH_REPOSITORY}:${t}-arm64")
+          done
+
+          LABELS=(
+            --label "org.label-schema.schema-version=1.0"
+            --label "org.label-schema.vcs-ref=$GITHUB_SHA"
+            --label "org.label-schema.vcs-url=$GITHUB_REPO_URL"
+            --label "org.label-schema.version=$VERSION"
+          )
```

Comment:
This is a quasi-standard for labels. See http://label-schema.org/. We will use the version label here in the release workflow to verify we are creating a release for the correct version.